### DISCUSSION TITLE
error: unrecognized command-line option '--auth=esp8266'

### DISCUSSION THREAD

**Discussion by AttiMM** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#425] [General]
Hi All,
did you experience this error when trying to upload firmware via OTA?

After I uncompressed WiFi-remote-for-Bestway-Lay-Z-SPA-3.5.1.zip and opened \Code\6-wire-version\platformio.ini in VSC, I modified \lib\BWC\model.h according to my tub, and also the last lines of platform.ini for the OTA upload:
![image](https://user-images.githubusercontent.com/10764890/210019293-8987c364-1045-401e-a0f1-f254ee2111c3.png)

Then I used the right arrow at bottom-left to Upload the firmware via OTA according to the given parameters in platform.ini, but it's failed because of this error:
![image](https://user-images.githubusercontent.com/10764890/210019416-e42ab712-4249-4841-b1e0-1149da801206.png)
![image](https://user-images.githubusercontent.com/10764890/210019534-38b880f5-c96d-4887-875e-1f87e59219cf.png)

**Steps that not helped:**
- I've got the same if I only Build the firmware without Uploading
- restart VSC
- PlatformIO / Project tasks / Clean All
- restart computer

**Does anybody have any idea how to resolve this issue? The tub is in use so I don't want to open the house of pump for uploading via USB cable if it's not absolutely necessary.**

Thanks for your help

**Comment by visualapproach** [comment]
> Please try it again after you set up for OTA uploading. If I set up for USB uploading (commented out the lines the problematic lines) I can build (and upload via USB) it without any problem.

Uploaded OTA without problems...

**Comment by rosetown-kelly** [comment]
I think you just need to fix your indentation. You've got an extra space at the beginning of lines 43-46.

**Reply by torei** [reply]
yea looks like
another tip: use ctrl+# to comment/uncomment in vscode

**Reply by AttiMM** [reply]
Dear All, and especially @rosetown-kelly !

Thank you so much for your time and efforts to help me! Yes, the cause of the issue was one extra space in the problematic line!
Fixing this line I could build and upload firmware OTA with no problem!
I could upload file system as well, not from VSC with "Upload Filesystem Image OTA" command because of an error (_*** [uploadfsota] Error 1_) but from web interface with File Uploader (thanks for this hint, visualapproach!)

So, thanks again and I wish Happy new year for all of you and your family! :-)
